### PR TITLE
Updating Contract - Events committee

### PR DESCRIPTION
**Pull request by Tguntenaar**
The events-committee contract is out of date with all the latest changes made to `devhub.near`. 

The front-end is now on the `main` branch in the neardevhub-bos repo but doesn't deploy automatically because I purposly didn't add the vars/secret to the devhub.near github action environment yet.

In order to update the events-committee.near we need to update both front end and back end together to avoid breaking changes.

Also need to update the indexer of the events-committee.

### CODE REVIEW

File: src/access_control/members.rs [hunk 1]
```diff
@@ -245,7 +245,7 @@ mod tests {
 
     fn root_member() -> (Member, VersionedMemberMetadata) {
         (
-            Member::Account("devgovgigs.near".parse().unwrap()),
+            Member::Account("events-committee.near".parse().unwrap()),
```
Comment (frol) [inline src/access_control/members.rs:248]: Do we need this change?

File: src/proposal/mod.rs [hunk 2]
```diff
@@ -245,13 +245,14 @@ pub fn get_subscribers(proposal_body: &ProposalBodyV2) -> Vec<String> {
...
-        String::from("Tooling & Infrastructures"),
-        String::from("Other"),
+        String::from("Bounty"),
+        String::from("Bounty booster"),
+        String::from("Event sponsorship"),
+        String::from("Events team"),
+        String::from("Hackathon"),
+        String::from("Hackbox"),
+        String::from("Meetup"),
+        String::from("Travel expenses"),
```
Comment (frol) [inline src/proposal/mod.rs:255]: I think there is no sane defaults anymore, so I would drop the whole function completely.

File: src/proposal/repost.rs [hunk 3]
```diff
@@ -6,7 +6,8 @@ use devhub_common::social_db_contract;
 use crate::Proposal;
 
 pub fn proposal_repost_text(proposal: Proposal) -> String {
-    let proposal_link = format!("/devhub.near/widget/app?page=proposal&id={}", proposal.id);
+    let proposal_link =
+        format!("/events-committee.near/widget/app?page=proposal&id={}", proposal.id);
```
Comment (frol) [inline src/proposal/repost.rs:10]: It sounds like we should use `near_sdk::env::current_account_id()` here

File: tests/proposals.rs [hunk 4]
```diff
@@ -27,7 +27,7 @@ async fn test_proposal() -> anyhow::Result<()> {
                 "proposal_body_version": "V0",
                 "name": "another post",
                 "description": "some description",
-                "category": "Marketing",
+                "category": "Bounty",
```
Comment (frol) [inline tests/proposals.rs:30]: I would not change these. Instead, I would explicitly initialize the categories, so we don't need to rely on default categories list

**Review by frol — CHANGES_REQUESTED** [review]

**Comment by Tguntenaar** [comment]
@frol this PR isn't supposed to merge into the main branch. These are just the changes I made to quickly deploy the events-committee contract. I should have made an issue instead, so I will close it.

**Pull request closed without merging**